pkg/sources/generator/tickgen.go

```diff
@@ -215,13 +215,7 @@ func (mg *memgen) IsEmpty() bool {
 	return len(mg.srcchan) == 0
 }
 
-// Read reads a chunk of messages and returns at the first occurrence of an error. Error does not indicate that the
-// array of result is empty, the callee should process all the elements in the array even if the error is set. Read
-// will not mark the message in the buffer as "READ" if the read for that index is erring.
-// There is a chance that we have read the message and the container got forcefully terminated before processing. To provide
-// at-least-once semantics for reading, during restart we will have to reprocess all unacknowledged messages.
-//
-// the context passed to read should be different from the lifecycle context that is used by this vertex.
+// The context passed to read should be different from the lifecycle context that is used by this vertex.
```

Comment:
golang function comment should start with the function name.

Reply:
+1. Addressed by removing this piece of comment. The context passed to read is not used anyway.